docs/section5/Rule5-37.md

```diff
@@ -17,68 +17,66 @@
+          **Rule Assertion - Component:**
 
-            **Rule Assertion - Component:**
+          - Case 1； For each subsurface, if roof is exterior mixed type and the baseline requirements for residential and non-residential type U-factor for different skylight-roof-ratio are different: `if manual_review_flag AND ( scc_dictionary_b[surface_b] == "EXTERIOR MIXED" ): UNDETERMINED`
 
-            - Case 1； For each subsurface, if roof is exterior mixed type and the baseline requirements for residential and non-residential type U-factor for different skylight-roof-ratio are different: `if manual_review_flag AND ( scc_dictionary_b[surface_b] == "EXTERIOR MIXED" ): UNDETERMINED`
+          - Case 2: Else if roof is exterior mixed type and skylight U-factor matches Table G3.4 requirement: `else if ( scc_dictionary_b[surface_b] == "EXTERIOR MIXED" ) AND ( subsurface_b.u_factor == target_u_factor_mixed ): PASS`
 
-            - Case 2: Else if roof is exterior mixed type and skylight U-factor matches Table G3.4 requirement: `else if ( scc_dictionary_b[surface_b] == "EXTERIOR MIXED" ) AND ( subsurface_b.u_factor == target_u_factor_mixed ): PASS`
+          - Case 3: Else if roof is exterior residential type and skylight U-factor matches Table G3.4 requirement: `else if ( scc_dictionary_b[surface_b] == "EXTERIOR RESIDENTIAL" ) AND ( subsurface_b.u_factor == target_u_factor_res ): PASS`
 
-            - Case 3: Else if roof is exterior residential type and skylight U-factor matches Table G3.4 requirement: `else if ( scc_dictionary_b[surface_b] == "EXTERIOR RESIDENTIAL" ) AND ( subsurface_b.u_factor == target_u_factor_res ): PASS`
+          - Case 4: Else if roof is exterior non-residential type and skylight U-factor matches Table G3.4 requirement: `else if ( scc_dictionary_b[surface_b] == "EXTERIOR NON-RESIDENTIAL" ) AND ( subsurface_b.u_factor == target_u_factor_nonres ): PASS`
 
-            - Case 4: Else if roof is exterior non-residential type and skylight U-factor matches Table G3.4 requirement: `else if ( scc_dictionary_b[surface_b] == "EXTERIOR NON-RESIDENTIAL" ) AND ( subsurface_b.u_factor == target_u_factor_nonres ): PASS`
+          - Case 5: Else if roof is semi-exterior type and skylight U-factor matches Table G3.4 requirement: `else if ( scc_dictionary_b[surface_b] == "SEMI-EXTERIOR" ) AND ( subsurface_b.u_factor == target_u_factor_semiheated ): PASS`
 
-            - Case 5: Else if roof is semi-exterior type and skylight U-factor matches Table G3.4 requirement: `else if ( scc_dictionary_b[surface_b] == "SEMI-EXTERIOR" ) AND ( subsurface_b.u_factor == target_u_factor_semiheated ): PASS`
-
-            - Case 6: Else: `else: FAIL`
+          - Case 6: Else: `else: FAIL`
 
 **Rule Assertion - RMR:**
```

Comment:
I don't think we should require manual review of all surfaces if there is at least one Mixed surface that triggers Undertermined outcome.

Reply:
Discussed at the RDS call, if one surface triggers a flag as mixed (res/non res) type, we will not be able to determine the actual res or non-res window-wall-ratio percentage, so the entire rule will be tagged as "undetermined".